typings/index.d.ts

```diff
@@ -1167,6 +1167,7 @@ declare module 'discord.js' {
 
     public count: number | null;
     public readonly emoji: GuildEmoji | ReactionEmoji;
+    public readonly client: Client;
```

Comment:
this should be above `count`, to be sorted alphabetically

Reply:
No worries, I'll update in the morning :)